kitsune/users/views.py

```diff
@@ -772,3 +776,41 @@ def validate_field(request):
                 }
 
     return data
+
+
+class FXAAuthenticateView(OIDCAuthenticationRequestView):
+
+    @staticmethod
+    def get_settings(attr, *args):
+        """Override settings for Firefox Accounts.
+
+        The default values for the OIDC lib are used for the SSO login in the admin
+        interface. For Firefox Accounts we need to pass different values, pointing to the
+        correct endpoints and RP specific attributes.
+        """
+
+        FXA_CONF = {
+            'OIDC_OP_TOKEN_ENDPOINT': settings.FXA_OP_TOKEN_ENDPOINT,
+            'OIDC_OP_AUTHORIZATION_ENDPOINT': settings.FXA_OP_AUTHORIZATION_ENDPOINT,
+            'OIDC_OP_USER_ENDPOINT': settings.FXA_OP_USER_ENDPOINT,
+            'OIDC_RP_CLIENT_ID': settings.FXA_RP_CLIENT_ID,
+            'OIDC_RP_CLIENT_SECRET': settings.FXA_RP_CLIENT_SECRET,
+            'OIDC_AUTHENTICATION_CALLBACK_URL': 'users.fxa_authentication_callback',
+            'OIDC_RP_SCOPES': settings.FXA_RP_SCOPES,
+            'OIDC_USE_NONCE': settings.FXA_USE_NONCE
+        }
+
+        if attr in FXA_CONF:
+            return FXA_CONF[attr]
+        return super(FXAAuthenticateView, FXAAuthenticateView).get_settings(attr, *args)
+
+
+class FXALogoutView(OIDCLogoutView):
+
+    @staticmethod
+    def get_settings(attr, *args):
+        """Override the logout url for Firefox Accounts."""
+
+        if attr == 'LOGOUT_REDIRECT_URL':
```

Comment:
Same here. I would prefer to have a function that imports all the settings instead just for consistency. To avoid any future errors where we forget to change a config in one of the 3 `get_settings`